src/components/DayPickerNavigation.jsx

```diff
@@ -80,6 +91,7 @@ export default function DayPickerNavigation(props) {
       }
 
       <span
+        aria-label={phrases.jumpToNextMonth}
```

Comment:
Yes. I figured we'd do some of the small aria-labeling/add those phrases unrelated to keyboard nav here.